src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -54,6 +54,19 @@ define([
                         var saveCode = $(e.target).data('saveCode');
                         self.plugin.turnOff();
                         self.triggerEvent('launchpad:activate-scenario', saveCode);
+                    })
+                    .on('click', 'button.truncate-toggle', function (e) {
+                        var parentEl = $(e.target.parentElement);
+                        $('p', parentEl).each(function() {
+                            $(this).toggleClass('truncate-box');
+                        });
+                        $('.truncate-toggle', parentEl).each(function() {
+                            if ($(this).text() === 'View more') {
+                                $(this).text('View less');
+                            } else {
+                                $(this).text('View more');
+                            }
```

Comment:
I'm not sure how literal the blurb in #749 was intended to be, but again I'd think and ellipsis and/or arrow icons would be sufficient.